Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12-slim
+LABEL maintainer="denys.kotovskiy@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
Consider copying only the `requirements.txt` file first and installing dependencies before copying the rest of the application files. This can help leverage Docker's caching mechanism to avoid reinstalling dependencies if only application code changes.